lib/s3_writer.rb

```diff
@@ -50,6 +55,8 @@ def feed_xml(checkouts)
             xml['dc'].contributor checkout.author if checkout.has? :author
             xml['dc'].identifier "urn:isbn:#{checkout.isbn}" if checkout.has? :isbn
             xml['dc'].identifier "urn:barcode:#{checkout.barcode}" if checkout.has? :barcode
+            xml.collection checkout.collection
+            xml.coarse_item_type checkout.coarse_item_type
```

Comment:
Can this be lower-camel-case?

Also, are we adding the checkout indexes to each entry?

Reply:
I've camel-cased these and added checkout indices.